packages/core/src/utils.ts

```diff
@@ -381,21 +381,22 @@ export function toObserver<T>(
 }
 
 export function createInvokeId(stateNodeId: string, index: number): string {
-  return `${index}.#${stateNodeId}`;
+  return `${index}.${stateNodeId}`;
```

Comment:
feel free to revert this part of the change. It just felt that `#` is also a special character that we don't necessarily have to include here.